android/app/src/main/java/org/fossasia/openevent/core/notifications/NotificationsFragment.java

```diff
@@ -116,8 +117,8 @@ private void clearViews() {
     public void onNotificationsDownloadDone(boolean status) {
         if (!status) {
             Timber.d("Notifications download failed");
-            if (getActivity() != null && swipeRefreshLayout != null) {
-                Snackbar.make(swipeRefreshLayout, getActivity().getString(R.string.refresh_failed), Snackbar.LENGTH_LONG).setAction(R.string.retry_download, view -> refresh()).show();
+            if (getActivity() != null && swipeRefreshLayout != null ) {
```

Comment:
@simarsingh24 There is an unwanted space after null :+1:

Reply:
removed :+1